src/components/SetOwnerModal.tsx

```diff
@@ -76,7 +74,7 @@ const AddOwnerModalContent = ({ onClose }: { onClose: () => void }) => {
       }}
     >
       <Form>
-        <Text variant="featured-2">Add Owner</Text>
+        <Text variant="featured-2">Add Signer</Text>
```

Comment:
Should the other variables be changed to signer too? noticed they all still are called owner